ut/diter.cg

```diff
@@ -13,7 +13,7 @@
     services=[service-0, service-1, service-2, service-3, service-4, service-5,
               service-6, service-7])
 (service-0 type=@M0_CST_IOS endpoints=["0@lo:12345:34:1"] params=[]
-    sdevs=[sdev-10, sdev-11, sdev-12, sdev-13, sdev-14])
+    sdevs=[sdev-10, sdev-11, sdev-12, sdev-13])
```

Comment:
Changes in diter.cg need to be removed; are for optional testing; just revert this file

Reply:
reverted file.
commit be43731edd92782fb063b3f9e3d72d4cc078be29